src/api/store/misc.py

```diff
@@ -271,3 +272,38 @@ def backfill_tag_data(self, context: Context, args):
     except Exception as e:
       capture_message(str(e), level="error")
       return None, CustomMassenergizeError(e)
+  
+  
+  def create_carbon_equivalency(self, args):
+    try:
+      new_carbon_equivalency = CarbonEquivalency.objects.create(**args)
```

Comment:
Probably needs a change relating to the  icon ForeignKey.  See create_community, how logo is popped out of the args and then added after the create(**args).

Reply:
I've written this using what I wrote for the update method.
`def create_carbon_equivalency(self, args):
    try:
      new_carbon_equivalency = CarbonEquivalency.objects.create(**args)

      value = args.get('value', None)
      if value:
        new_carbon_equivalency.value = value

      explanation = args.get('explanation', None)
      if explanation:
        new_carbon_equivalency.explanation = explanation

      reference = args.get('reference', None)
      if reference:
        new_carbon_equivalency.reference = reference

      icon = args.get("icon", None)
      if icon:
        cFav = Media(file=icon, name=f"{args.get('name', '')} CarbonEquivalencyFavicon")
        cFav.save()
        new_carbon_equivalency.icon = cFav

      new_carbon_equivalency.save()
      
      return new_carbon_equivalency, None

    except Exception as e:
      capture_message(str(e), level="error")
      return None, CustomMassenergizeError(e)`